components/hab/src/error.rs

```diff
@@ -48,6 +48,7 @@ pub enum Error {
     HabitatCore(hcore::Error),
     // Boxed due to clippy::large_enum_variant
     HandlebarsRenderError(Box<handlebars::TemplateRenderError>),
+    InvalidUrl(String),
```

Comment:
I dont think we should add this variant. See other comments.

Reply:
Variant removed in https://github.com/habitat-sh/habitat/pull/7841/commits/78f9d5e6a2b76e641bb6f2a2971c62744b5e762b